### ISSUE TITLE
setup: display additional information when doing a test send

### ISSUE THREAD

**Issue by billygarrison** [jamesob/coldcore#32]
The Coldcard displays

output 0 address (change)
output 1 address

And I think total output amount (input - fee)

It would be easier to visually verify transaction details on Coldcard if coldcore displayed the same info in the same order.

**Comment by billygarrison** [comment]
I can make this change

**Comment by billygarrison** [comment]
Example Coldcard output:
```
OK TO SEND?
Consolidating
0.00099859 XTN
within wallet.

1 ins - fee
 = 2 outs

Network fee:
0.00000141 XTN

Change back:
0.00099859 XTN
 - to addresses -
tb1qhl02...lm5
tb1q7d2...sju

Press OK to approve and sign transaction. X to abort.
```

**Comment by billygarrison** [comment]
Example Coldcore output:
![image](https://user-images.githubusercontent.com/30005204/110249372-3c3ffa80-7f4c-11eb-8488-d4cd563c8cf4.png)

**Comment by jamesob** [comment]
Yeah, agree it would be better to display both the total value and the principal/fee decomposition to allow easier verification on the coldcard.
